src/Security/Member.php

```diff
@@ -1705,6 +1705,16 @@ public function getHtmlEditorConfigForCMS()
         $currentName = '';
         $currentPriority = 0;
 
+        // If we don't have a custom config, no need to look in all groups
+        $editorConfigMap = HTMLEditorConfig::get_available_configs_map();
+        $editorConfigCount = count($editorConfigMap);
+        if ($editorConfigCount === 0) {
+            return 'cms';
+        }
```

Comment:
Why do we default to "cms" in this scenario? If there's no available config maps what would we expect returning "cms" to do here?

Reply:
because that's the current default if no value is found ?
this method always return something, "cms" being the default.

Reply:
Is the default defined somewhere? A config?

Reply:
It's hardcoded currently https://github.com/silverstripe/silverstripe-framework/blob/9d1741cecd368f5ed277c2150e692b04f1084abe/src/Security/Member.php#L1721

i kept it the way it was :-)

Reply:
That makes sense - that part of code was hidden in the diff and I didn't think to expand to see further down.